benchmark/single-source/AngryPhonebook.swift

```diff
@@ -39,3 +49,39 @@ public func run_AngryPhonebook(_ N: Int) {
     }
   }
 }
+
```

Comment:
Set the workload baseline by the slowest benchmark to be under 1000 μs, looking at the last measured performance of `Armenian` (27^2 = 2636 name pairs), I'm guessing we could standardize on workload size of 20 names per script. We should also try to keep the overall size roughly the same. Document this:
````swift
// Workloads for various scripts. Always 20 names for 400 pairings.
// To keep the performance of various scripts roughly comparable, aim for
// a total length of approximately 120 characters.
// E.g.: `latin.joined(separator: "").count == 124`

let latin = Array(words.prefix(20))

let cyrillic = [
  "Александр", "Аркадий", "Аня", "Даниил", "Дмитрий", "Эдуард", "Юрій", "Давид",
  "Анна", "Дмитрий", "Евгений", "Борис", "Владимир", "Артур", "Антон",
  "Антон", "Надія", "Алёна", "Алиса"]

let armenian = [
  "Արմեն", "Աննա", "Հարություն", "Միքայել", "Մարիա", "Դավիթ", "Վարդան", "Նարինե",
  "Հռիփսիմե", "Տիգրան", "Տաթև", "Ադամ", "Ազատ", "Ազնաւուր", "Գրիգոր", "Անի",
  "Լիլիթ", "Հայկ", "Անդրանիկ", "Գառնիկ"]
````
I think `cyrillic` needs a bit tuning, the first 20 names I took are only 108 characters… could you please bring it up to ~120? Armenian at 114 are fine, I think…

Reply:
Armenian and Cyrillic are both 2-byte encoded languages. Unless there are salient differences in how capitalization works (I don't know either enough to know), normalization, or use of combining characters, these would be the same workload from a technical perspective.

As @palimondo pointed out, what's also interesting would be the following:

let smallTwoByte = [
  // something up to 15 UTF-8 code units in length, i.e. 7 "characters" in Cyrillic or Armenian ...
]
let largeTwoByte = [
  // something > 16 UTF-8 code units in length, i.e. at least 7 "characters" in Cyrillic or Armenian...  
]

And the same for 3-byte for some latter-BMP language with capitalization (not sure off the top of my head), and without (e.g. Han ideographs), and similarly for 4-byte, and a large 1-byte (i.e. ASCII) workload. This can be done as a separate PR, though.